<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ap240verticalform</class>
 <widget class="QWidget" name="ap240verticalform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="2">
      <widget class="QComboBox" name="comboBox">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>0: no bandwidth limit</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1: bandwidth limit at 25MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2: bandwidth limit at 700MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3: bandwidth limit at 200MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4: bandwidth limit at 20MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5: bandwidth limit at 35MHz</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Band width</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Offset (V)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Full Scale (V)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox">
       <property name="minimum">
        <double>-5.000000000000000</double>
       </property>
       <property name="maximum">
        <double>5.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Coupling</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="comboBox_2">
       <property name="currentIndex">
        <number>3</number>
       </property>
       <item>
        <property name="text">
         <string>0 Ground (Averagers ONLY)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1: DC, 1Mohm</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2: AC, 1Mohm</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3: DC, 50ohm</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4: AC, 50ohm</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Invert signal</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="comboBox_3">
       <item>
        <property name="text">
         <string>5.0V</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2.0V</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.0V</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.5V</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.2V</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.1V</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.05V</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
